main-app/app/tests/core/authenticate/bingo-authentication-proxy.test.js

```diff
@@ -0,0 +1,43 @@
+(function () {
+    'use strict';
+    describe('Test the API wrapper service', function () {
+        var $httpBackend,
+            proxy;
+
+        beforeEach(function () {
+            module('Tombola.BingoClient', function($provide){
+                $provide.value(mocks.bingoApiProxy, 'call');
+                $provide.value(mocks.userAuthenticationUpdater);
+            });
+            inject(function ($injector) {
+                $httpBackend = $injector.get('$httpBackend');
+                proxy = $injector.get('BingoAuthenticationProxy');
+            });
+        });
+
+        it('The Api service responds with the correct value in a login request', function () {
+            $httpBackend
+                .expectPOST('http://localhost:30069/users/login', {"username":"drwho", "password":"tardis123!"})
+                .respond( {
+                    "message": "LoginSuccess",
+                    "payload": {
+                        "user": {
+                            "username": "drwho",
+                            "balance": 20000,
+                            "token": "response"
+                        }
+                    }
+                });
+            var returnPromise = proxy.makeLoginRequest("drwho", "tardis123!");
+            returnPromise.then(function (response) {
+                response.should.equal('response');
+            });
+            $httpBackend.flush();
+        });
+
+        afterEach(function () {
+            $httpBackend.verifyNoOutstandingExpectation();
+            $httpBackend.verifyNoOutstandingRequest();
+        });
+    });
+})();
```

Comment:
Putting this at the end as it there is nowhere else to put this as it is something [potentially] missing. Given the service uses both `bingoApiProxy` and `userAuthenticationUpdater` and given the job the service it is doing -  the relevant methods being called on the dependencies are *probably* essential to the proper functioning of the service under test. If this is the case then you need to ensure those functions are being called with the correct values - using a spy.